tests/integrated/test-region-iterator/runtest

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+#
+# Run the test, check it completed successfully
+#
+
+from __future__ import print_function
+try:
+  from builtins import str
+except:
+  pass
+from boututils.run_wrapper import shell, shell_safe, launch_safe, getmpirun
+from boutdata.collect import collect
+from sys import exit
+
+MPIRUN=getmpirun()
+
+print("Making Region Iterator test")
+s, out = shell_safe("make > make.log",pipe=True)
+
+flags = [""]
+cmd = "./test_region_iterator"
+code = 0 # Return code
+pipe=True
+
+for nproc in [1,2]:#Number of mpi procs
+  for mthread in [1]:#Number of omp threads (not yet supported)
+    print("\t{n} processors and {m} threads".format(n=nproc,m=mthread))
+
+    for f in flags:
+        #print("\tflags '"+f+"'")
```

Comment:
Not sure if it's just github, but it looks like you're using 4 spaces for indentation here, but 2 elsewhere

Reply:
further `flags` is not used and `f` is overwritten - sounds just like what I fixed in #843 :)